ResearchKit/Common/ORKPasscodeViewController.h

```diff
@@ -66,6 +67,24 @@ ORK_AVAILABLE_DECL
  */
 - (void)passcodeViewControllerDidCancel:(UIViewController *)viewController;
 
+/*
+ * Defaults to Localized "Forgot Password?" text
+ * @return the text of the forgot passcode button
+ */
+- (NSString*)passcodeViewControllerTextForForgotPasscode:(UIViewController *)viewController;
+
+/*
+ * Defaults to Localized standard Apple blue tint color
+ * @return the tint color of the forgot passcode button
+ */
+- (UIColor*)passcodeViewControllerTintColorForForgotPasscode:(UIViewController *)viewController;
```

Comment:
@YuanZhu-apple I don't remember. I looked in the PRs for this (because I remember talking to the dev who implemented this about doing that), but I cannot find a reference to it so I am guessing it was in an in-person conversation. My guess is that this was a time constraint issue b/c there was a push to have this feature for release of mPower 1.3 and that was constrained by the March keynote.  I'll rewrite it.